libraries/chain/db_market.cpp

```diff
@@ -932,7 +932,11 @@ bool database::fill_settle_order( const force_settlement_object& settle, const a
 { try {
    bool filled = false;
 
-   auto issuer_fees = pay_market_fees(get(receives.asset_id), receives);
+   const account_object& owner = settle.owner(*this);
+
+   auto issuer_fees = ( head_block_time() < HARDFORK_CORE_1780_TIME ) ?
+      pay_market_fees(get(receives.asset_id), receives) :
+      pay_market_fees(owner, get(receives.asset_id), receives);
```

Comment:
We can avoid the local `owner` variable for better performance before HF.

Reply:
Sounds reasonable. Thank you. 
Done